### DISCUSSION TITLE
tflite files for download?

### DISCUSSION THREAD

**Discussion by Triffid-ger** [jomjol/AI-on-the-edge-device#3435] [Q&A] [answered]
Hi there!
For my camera setup, the "s2" size-files of the tflite-files seem to be too small. I would like to try s1 files, but I cannot find a place to download any tflite-Files?!
Am I missing something?
Greetings
Sebastian

**Comment by SybexX** [comment, marked as answer]
The tflite-files are included in the zip files of the firmware https://github.com/jomjol/AI-on-the-edge-device/releases or
https://github.com/jomjol/neural-network-digital-counter-readout
https://github.com/jomjol/neural-network-analog-needle-readout

**Comment by Triffid-ger** [comment]
Ah, thank you. I see. I have seen those before but forgot about it. Unfortunately there don't seem to be any s1-Versions. I have some, though, which worked quite well on another meter but those files are pretty old.
So I guess I simply move the camera out a little, if space allows.
